taxcalc/tests/pufcsv_agg_expect.txt

```diff
@@ -1,20 +1,22 @@
+                                            2013     2014     2015     2016     2017     2018     2019     2020     2021     2022
+Returns (#m)                               163.1    162.8    164.9    167.6    170.4    173.2    175.8    177.8    180.1    182.4
+AGI ($b)                                 9,326.3 10,020.2 10,544.9 11,022.6 11,443.9 11,874.5 12,280.3 12,720.4 13,221.5 13,765.9
+Itemizers (#m)                              42.3     42.1     42.9     44.1     45.0     45.7     46.3     47.1     47.7     48.2
+Itemized Deduction ($b)                  1,094.7  1,107.8  1,161.8  1,221.7  1,274.8  1,336.1  1,397.5  1,464.3  1,533.9  1,606.9
+Standard Deduction Filers (#m)             108.3    108.0    109.0    110.2    111.8    113.5    115.0    116.2    117.5    119.0
+Standard Deduction ($b)                    929.4    936.9    962.4    973.2    995.5  1,033.2  1,071.9  1,109.1  1,149.1  1,190.8
+Personal Exemption ($b)                  1,117.5  1,116.6  1,139.5  1,164.6  1,176.3  1,215.0  1,254.9  1,295.7  1,337.2  1,379.7
+Taxable Income ($b)                      6,639.0  7,329.6  7,758.4  8,146.0  8,483.7  8,792.5  9,074.8  9,377.7  9,744.7 10,153.3
+Regular Tax ($b)                         1,453.7  1,667.6  1,782.1  1,877.0  1,949.4  2,010.0  2,065.2  2,126.7  2,207.1  2,300.1
+AMT Income ($b)                          8,784.1  9,483.0  9,979.4 10,425.7 10,821.1 11,221.3 11,596.3 12,000.7 12,467.2 12,976.1
+AMT Liability ($b)                          36.7     38.8     40.4     42.8     45.1     47.3     49.7     51.7     54.5     57.8
+AMT Filers (#m)                              4.5      4.8      5.0      5.3      5.6      5.7      5.8      5.9      6.1      6.2
+Tax before Credits ($b)                  1,389.7  1,565.7  1,658.1  1,742.1  1,815.1  1,876.6  1,933.7  1,993.9  2,072.1  2,162.2
+Refundable Credits ($b)                     99.5     97.6     97.7     96.8     96.4     98.2    100.3    102.1    104.0    105.8
+Nonrefundable Credits ($b)                  72.9     73.4     73.7     73.9     74.7     75.1     75.5     76.1     76.7     77.8
+Reform Surtaxes ($b)                         0.0      0.0      0.0      0.0      0.0      0.0      0.0      0.0      0.0      0.0
+Ind Income Tax ($b)                      1,246.4  1,433.1  1,530.6  1,618.5  1,691.9  1,751.7  1,806.8  1,866.0  1,943.2  2,032.6
+Payroll Taxes ($b)                         924.4    958.3    999.5  1,039.7  1,093.5  1,140.7  1,184.3  1,229.9  1,278.4  1,329.9
+Combined Liability ($b)                  2,170.8  2,391.3  2,530.1  2,658.2  2,785.4  2,892.4  2,991.1  3,095.9  3,221.7  3,362.5
+Non-Positive Income Tax Liability (#m)      70.8     70.3     70.4     70.5     70.6     71.5     72.5     72.9     73.6     74.3
+Non-Positive Payroll Tax Liability (#m)     36.8     38.0     38.9     40.1     41.2     42.4     43.7     44.4     45.5     46.8
```

Comment:
Let's do "Non-Positive Combined Tax Liability" instead of non-positive payroll tax liability..